src/SentryLink.ts

```diff
@@ -67,9 +67,9 @@ export class SentryLink extends ApolloLink {
             if (
               this.options.attachBreadcrumbs.includeError &&
               result.errors &&
-              result.errors.length > 1
+              result.errors.length > 0
             ) {
-              // We must have a breadcrumb if attachBreadcrumbs was set
+              // Include any errors if the option is set
```

Comment:
That comment should stay as is, it provides a reason why a type assertion is used here. I also think your new comment does not add value, as it simply reiterates what the code does.

Reply:
Oh okay.

You could remove the typecast if you change all instance of:
```
if (this.options.attachBreadcrumbs) {
```
to
```
if (breadcrumb) {
```

Reply:
Changing the checks as you suggest simply shifts to TypeScript complaining about unsafe accesses to `this.options.attachBreadcrumbs` instead. TypeScript does not understand that `breadcrumb` and `this.options.attachBreadcumbs` are related.

Checking for both every time would eliminate the need for type assertions, but introduce unnecessary runtime checks.